src/main/java/io/strimzi/kafka/metrics/DataPointSnapshotBuilder.java

```diff
@@ -12,20 +12,30 @@
 import io.prometheus.metrics.model.snapshots.Quantiles;
 import io.prometheus.metrics.model.snapshots.SummarySnapshot;
 
+import java.util.logging.Logger;
```

Comment:
We should use `org.slf4j.Logger` instead of `java.util.logging.Logger`

Reply:
That's why I couldn't use interpolation with `{}`. I was using the java.util logger. Changed to `sl4j`